web/modules/custom/joinup_sparql/src/Database/Driver/sparql/Connection.php

```diff
@@ -28,7 +28,7 @@ public function query(string $query, array $args = [], array $options = []): Res
       // reasonable amount of time and then we retry one more time.
       // @see http://docs.openlinksw.com/virtuoso/checkpoint/
       sleep(5);
-      return $this->query($query);
```

Comment:
I wonder how I approved this? :) But my proposal in that ticket was OK just I didn't made my duty on the 2nd round of QA